kokoro/gcp_ubuntu/continuous.cfg

```diff
@@ -0,0 +1,7 @@
+# -*- protobuffer -*-
+# proto-file: google3/devtools/kokoro/config/proto/build.proto
+# proto-message: BuildConfig
+
+# Location of the bash script. Should have value <github_scm.name>/<path_from_repository_root>.
+# github_scm.name is specified in the job configuration.
+build_file: "fhir-access-proxy/kokoro/gcp_ubuntu/kokoro_build.sh"
```

Comment:
Do we get a distinct image tag when this is run in the continuous mode? Or is the same `BUILD_ID` used as in the PR case? I guess I am wondering what would be `${KOKORO_BUILD_ID}` in both cases.

Reply:
It is fine to have a `latest` tag but it would also be nice to first introduce a `release` tag and make sure the one tagged as `release` is well-tested (e.g., manually too).